src/Microsoft.ML.LightGBM.StaticPipe/LightGbmStaticExtensions.cs

```diff
@@ -122,11 +122,13 @@ public static class LightGbmStaticExtensions
         /// ]]></format>
         /// </example>
         public static (Scalar<float> score, Scalar<float> probability, Scalar<bool> predictedLabel) LightGbm(this BinaryClassificationCatalog.BinaryClassificationTrainers catalog,
-            Scalar<bool> label, Vector<float> features, Scalar<float> weights = null,
+            Scalar<bool> label,
+            Vector<float> features,
+            Scalar<float> weights = null,
             int? numLeaves = null,
             int? minDataPerLeaf = null,
             double? learningRate = null,
-            int numBoostRound = Options.Defaults.NumBoostRound,
+            int numBoostRound = Options.Defaults.NumberOfIterations,
```

Comment:
```suggestion
            int numberOfBoostRound = Options.Defaults.NumberOfIterations,
```

Reply:
resolved to numberOfIterations

---
In reply to: [261458031](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261458031) [](ancestors = 261458031)